app/models/manageiq/providers/openshift/container_manager_mixin.rb

```diff
@@ -32,10 +32,10 @@ def raw_connect(hostname, port, options)
     end
```

Comment:
Do you mean we could use '/oapi/v1/something' wherever we're using '/api/v1/something',
or that the new Kubeclient way to access '/oapi' is compatible to the old OpenshiftClient way to access '/oapi'?
(this PR deals with the latter, I was wondering here about the former in how Openshift Manager subclasses Kubernetes Manager)